### PR TITLE
Document --hypothesis-strategy CLI param

### PR DESCRIPTION
**Pull request by adamdabbracci**
Signed-off-by: Adam D'Abbracci <3945646+adamdabbracci@users.noreply.github.com>

### CODE REVIEW

File: sources/reference/usage/run.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ To test that you have a valid experiment you can pass the `--dry` option.
...
+
+* `default` 
+* `before-method-only`
+* `after-method-only`
+* `during-method-only`
+* `continously`
+
+For example:
+```
+chaos run ./experiment.json --hypothesis-strategy continously
```
Comment (ciaranevans) [inline sources/reference/usage/run.md:50]: Should be `continuously`

**Review by ciaranevans — CHANGES_REQUESTED** [review]
Just one more typo (sorry, I should have spotted this sooner!)

File: sources/reference/usage/run.md (hunk 1 above)
Comment (adamdabbracci) [reply sources/reference/usage/run.md]: I need to read more carefully... thanks :)

**Review by ciaranevans — APPROVED** [review]

**Comment by ciaranevans** [comment]
Huge thanks for the PR @adamdabbracci!

It's merged and available here https://chaostoolkit.org/reference/usage/run/#run-an-experiment-with-different-steady-state-strategies! 🎉

**Pull request merged**